### ISSUE TITLE
Q-Pagination Direction-Links Broken on Cordova

### ISSUE THREAD

**Issue by altShiftDev** [quasarframework/quasar#2088]
### Software version

Operating System        	Darwin(16.7.0) - darwin/x64
NodeJs                  	9.8.0

Global packages
  NPM                   	6.0.1
  yarn                  	0.24.6
  quasar-cli            	0.16.1
  vue-cli               	2.9.3
  cordova               	8.0.0

Important local packages
  quasar-cli            	0.16.1	(Quasar Framework CLI)
  quasar-framework      	0.16.0	(Build responsive websites, PWAs, hybrid mobile apps and Electron apps, all simultaneously using same codebase)
  quasar-extras         	2.0.1	(Quasar Framework fonts, icons and i18n.)
  vue                   	2.5.16	(Reactive, component-oriented view layer for modern web interfaces.)
  vue-router            	3.0.1	(Official router for Vue.js 2)
  vuex                  	3.0.1	(state management for Vue.js)
  electron              	Not installed
  electron-packager     	Not installed
  electron-builder      	Not installed
  @babel/core           	7.0.0-beta.46	(Babel compiler core.)
  webpack               	4.8.3	(Packs CommonJs/AMD modules for the browser. Allows to split your codebase into multiple bundles, which can be loaded on demand. Support loaders to preprocess files, i.e. json, jsx, es7, css, less, ... and your custom stuff.)
  webpack-dev-server    	3.1.4	(Serves a webpack app. Updates the browser on changes.)
  workbox-webpack-plugin	3.2.0	(A plugin for your Webpack build process, helping you generate a manifest of local files that workbox-sw should precache.)

iOS: iPhone X - 11.2 (Simulator)

Any other software related to your bug:
"cordova-ios": "~4.5.4"

### What did you get as the error?
No response when you use the next or prev arrows in q-pagination on iOS Cordova but working in Android Cordova. Sometime however on either platform the pages will keep incrementing after you press > and it will never stop until the system crashes.

### What were you expecting?
The page to increment by 1.

### What steps did you take, to get the error?
Here's my q-pagination, although you don't need anything this fancy to reproduce:
```
    <q-layout-footer v-model="footer">
      <q-pagination
        boundary-links
        direction-links
        input
        class="justify-around"
        text-color="black"
        v-model="page"
        :min="1"
        :max="maxPages" />
    </q-layout-footer>
```
In particular it seems the direction-links are the problem, they don't respond to click events. The boundary-links work fine though.

I've also experienced a difficult to reproduce bug that happens with the direction-links which causes the page to increment by 1 constantly, eventually the system will run out of ram and die but you can't stop it short of force-quitting the app, I've seen it happen in both iOS and Android, probably on web too.

**Comment by rstoenescu** [comment]
Hi,

I think you are referring to a feature where if you keep pressing the buttons it will continue to navigate (until you lift your finger off of the button).
Can't reproduce your bug... but regardless of that, now that I think of it, it's a bad thing to keep updating the Vue model since this generates a render for each model update and on mobile phones, this can degrade performance until user lifts the finger, then some delay until all renders are executed.

So pushed a fix to eliminate this feature. Available in future Quasar v0.16.1.

**Comment by altShiftDev** [comment]
@rstoenescu here's a video showing the behavior: https://streamable.com/cip22
